### PR TITLE
Fix descriptions not showing for theme properties

### PR DESCRIPTION
**Pull request by ajreckof**
Supersedes #75552

### CODE REVIEW

File: editor/editor_inspector.cpp [hunk 1]
```diff
@@ -3133,6 +3133,11 @@ void EditorInspector::update_tree() {
 			StringName propname = property_prefix + p.name;
 			bool found = false;
 
+			// Small hack for theme_overrides. They are listed under Control, but come from another class.
+			if (classname == "Control" && p.name.contains("theme_override")) {
```
Comment (YuriSizov) [inline editor/editor_inspector.cpp:3137]:
```suggestion
			if (classname == "Control" && p.name.begins_with("theme_override_")) {
```

**Comment by YuriSizov** [comment]
The trailing underscore was meaningful, as it ensures this is not the entire name.

**Comment by ajreckof** [comment]
Sorry I didn't see it. it has been corrected

**Review by akien-mga — APPROVED** [review]

**Review by KoBeWi — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.0.3.

**Pull request merged**